http/graphql/src/main/java/io/quarkus/ts/http/graphql/PersonsReactiveEndpoint.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.ts.http.graphql;
+
+import org.eclipse.microprofile.graphql.DefaultValue;
+import org.eclipse.microprofile.graphql.GraphQLApi;
+import org.eclipse.microprofile.graphql.Mutation;
+import org.eclipse.microprofile.graphql.Name;
+import org.eclipse.microprofile.graphql.Query;
+
+import io.smallrye.graphql.api.Context;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+
+@GraphQLApi
+public class PersonsReactiveEndpoint extends PersonsEndpointBase {
```

Comment:
Just curious: is there an option to have separate endpoints?

Reply:
In general, yes. But I havent found support in Quarkus/Smallrye.